app/routes/conference/index.js

```diff
@@ -15,5 +15,13 @@ export default Ember.Route.extend({
 		  link.href = uri;
 		  link.click();
 		},
+		deleteSubmission(conferenceId, submissionId) {
```

Comment:
https://guides.emberjs.com/v2.7.0/models/creating-updating-and-deleting-records/#toc_deleting-records